frontend/src/components/GastoRow.js

```diff
@@ -1,30 +1,58 @@
-import React from 'react';
+import React from "react";
 
-class GastoRow extends React.Component{
-  constructor(props){
+class GastoRow extends React.Component {
+  constructor(props) {
     super(props);
-     this.selectGasto=this.selectGasto.bind(this);
-    
+    this.selectGasto = this.selectGasto.bind(this);
+    this.actualizar = this.actualizar.bind(this);
```

Comment:
Esto creo que no es necesario, porque no lo llamás directamente desde la vista sino a través de otro método que ya tiene el `this` bien puesto.